security/security-release-process.md

```diff
@@ -0,0 +1,114 @@
+# Security Release Process
+
+etcd is a growing community of volunteers, users, and vendors. The etcd community has adopted this security disclosures and response policy to ensure we responsibly handle critical issues.
+
+## Product Security Committee (PSC)
+
+Security vulnerabilities should be handled quickly and sometimes privately. The primary goal of this process is to reduce the total time users are vulnerable to publicly known exploits.
+
+The PSC is responsible for organizing the entire response including internal communication and external disclosure but will need help from relevant developers and release leads to successfully run this process.
+
+The initial PSC will consist of volunteers who have been involved in the initial discussion:
+
+- Brandon Philips (**[@philips](https://github.com/philips)**) `<bphilips@redhat.com>` [4096R/154343260542DF34]
```

Comment:
I am happy to advise for a few months but would prefer others take over like @hexfusion